lib/livebook_web/live/session_live/shortcuts_component.ex

```diff
@@ -92,6 +92,7 @@ defmodule LivebookWeb.SessionLive.ShortcutsComponent do
       %{seq: ["N"], desc: "Insert Code cell above"},
       %{seq: ["M"], desc: "Insert Markdown cell above"},
       %{seq: ["z"], desc: "Toggle code zen"},
+      %{seq: ["o"], desc: "Toggle collapse section"},
```

Comment:
O is often tied to open. I would maybe go with `x` or `a`, since both are siblings to `z` (which is more presentation-like). My pick is `a` and we can use `cmd+shift+a` for collapse/expand all (cmd+a is select all on Mac). Thoughts?

Reply:
For the "all mode", the default is to collapse. We will only expand all once all sections are collapsed.

Reply:
```suggestion
      %{seq: ["o"], desc: "Expand/collapse section"},
```

Reply:
@josevalim we generally use cmd/shift/ctrl only in the editor (since there's no other way), so my suggestion would be to pick a common prefix, so we have `cs` (collapse section), `ca` (collapse all).

Reply:
There's also an option to use single letter and with shift it becomes "all".

Reply:
Alternate expand/collapse :P

Reply:
Otherwise I would go with "c" and "C" since collapse is the first action.

Reply:
I'm leaning towards "c" yeah, still close to "z".

Reply:
and zz for code zen

Reply:
Let's go with "c" and "C" then!